meteoritics-and-planetary-science.csl

```diff
@@ -125,8 +129,8 @@
   </citation>
   <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="1">
     <sort>
-      <key macro="author"/>
-      <key variable="title"/>
+      <key macro="author-short" names-use-first="1" names-min="2"/>
+      <key macro="year-date"/>
```

Comment:
The two behave differently, though and I'm not sure which one is correct: As is, this would sort

Smith, Bert 1776
Smith, Adam 1848
Smith, Bert, 1919

with `<key macro="author"/>` you'd get
Smith, Adam 1848
Smith, Bert 1776
Smith, Bert 1919